javascript/node/selenium-webdriver/lib/actions.js

```diff
@@ -125,213 +125,6 @@ class LegacyActionSequence {
     }
   }
 
-  /**
-   * Moves the mouse. The location to move to may be specified in terms of the
-   * mouse's current location, an offset relative to the top-left corner of an
-   * element, or an element (in which case the middle of the element is used).
-   *
-   * @param {(!./webdriver.WebElement|{x: number, y: number})} location The
-   *     location to drag to, as either another WebElement or an offset in
-   *     pixels.
-   * @param {{x: number, y: number}=} opt_offset If the target {@code location}
-   *     is defined as a {@link ./webdriver.WebElement}, this parameter defines
-   *     an offset within that element. The offset should be specified in pixels
-   *     relative to the top-left corner of the element's bounding box. If
-   *     omitted, the element's center will be used as the target offset.
-   * @return {!LegacyActionSequence} A self reference.
-   */
-  mouseMove(location, opt_offset) {
-    let cmd = new command.Command(command.Name.LEGACY_ACTION_MOUSE_MOVE)
-
-    if (typeof location.x === 'number') {
-      setOffset(/** @type {{x: number, y: number}} */ (location))
-    } else {
-      cmd.setParameter('element', location.getId())
-      if (opt_offset) {
-        setOffset(opt_offset)
-      }
-    }
-
-    this.schedule_('mouseMove', cmd)
-    return this
-
-    /** @param {{x: number, y: number}} offset The offset to use. */
-    function setOffset(offset) {
-      cmd.setParameter('xoffset', offset.x || 0)
-      cmd.setParameter('yoffset', offset.y || 0)
-    }
-  }
-
-  /**
```

Comment:
For this PR we were only looking at `mouseMove`. The other files may need tweaks where they call `mouseMove`